src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/ManagedTextSection.cs

```diff
@@ -147,14 +148,16 @@ public int CalculateOffsetToMappedFieldDataStream()
 
         internal int ComputeOffsetToDebugDirectory()
         {
-            Debug.Assert(MetadataSize % 4 == 0);
-            Debug.Assert(ResourceDataSize % 4 == 0);
+            Debug.Assert(MetadataSize % MetadataSizes.StreamAlignment == 0);
```

Comment:
Not 100% sure if this is the intent (4 byte alignment across the various sections); I think we could remove this Assert since they should all be aligned anyway